### PR TITLE
Whitelist password

### PR DESCRIPTION
**Pull request by leventdem**

### CODE REVIEW

File: src/components/PasswordStrength/PasswordStrength.module.scss [hunk 1]
```diff
@@ -1,5 +1,43 @@
...
+    position: relative;
+    top: 8px; 
+    left: 8px;
+  }
+
+  .InfoIcon {
+      position:relative;
+      top:3px;
+      left:3px
+    // padding-left: 5px;
```
Comment (G-Ray) [inline src/components/PasswordStrength/PasswordStrength.module.scss]: Extra comments

File: src/components/PasswordStrength/PasswordStrength.module.scss [hunk 2]
```diff
@@ -1,5 +1,43 @@
...
+    font-stretch: normal;
+    line-height: 1.23;
+    letter-spacing: normal;
+    color: #ffffff;
+    position: relative;
+    top: 8px; 
+    left: 8px;
+  }
+
+  .InfoIcon {
```
Comment (G-Ray) [inline src/components/PasswordStrength/PasswordStrength.module.scss:26]: I think a `cursor: pointer;` is missing

File: src/components/PasswordStrength/PasswordStrength.module.scss [hunk 3]
```diff
@@ -1,5 +1,43 @@
...
+    position: absolute;
+    z-index:2
+  }
+
+  .BubbleText {
+    width: 188px;
+    height: 38px;
+    font-family: Asap;
+    font-size: 13px;
+    font-weight: normal;
```
Comment (G-Ray) [inline src/components/PasswordStrength/PasswordStrength.module.scss]: you can remove all `nomal` properties

File: src/components/PasswordStrength/PasswordStrength.module.scss [hunk 4]
```diff
@@ -1,5 +1,43 @@
 @import '../../styles/mixins.scss';
 @import '../../styles/colors.scss';
+
+.Bubble {
+    width: 204px;
+    height: 53px;
+    border-radius: 3px;
+    box-shadow: 0 2px 12px 0 rgba(0, 0, 0, 0.2);
+    background-color: #5c6f84;
```
Comment (G-Ray) [inline src/components/PasswordStrength/PasswordStrength.module.scss]: This colors is defined in colors.scss file, you can use "$color-blue-grey"

File: src/components/PasswordStrength/PasswordStrength.module.scss (hunk 1 above)
Comment (leventdem) [reply src/components/PasswordStrength/PasswordStrength.module.scss]: ok, done

File: src/components/PasswordStrength/PasswordStrength.module.scss (hunk 3 above)
Comment (leventdem) [reply src/components/PasswordStrength/PasswordStrength.module.scss]: Removed.

File: src/components/PasswordStrength/PasswordStrength.module.scss (hunk 4 above)
Comment (leventdem) [reply src/components/PasswordStrength/PasswordStrength.module.scss]: ok use $color-blue-grey instead.

File: src/components/PasswordStrength/PasswordStrength.module.scss (hunk 2 above)
Comment (leventdem) [reply src/components/PasswordStrength/PasswordStrength.module.scss:26]: yes added, thanks.

File: src/components/PasswordStrength/PasswordStrength.jsx [hunk 5]
```diff
@@ -41,9 +41,13 @@ const InfoButton = () => {
...
   }
 
+  const handleMouseLeave = () => {
+    setVisible(false)
+  }
+
   return (
     <div className={styles.InfoIcon}>
-      <Info onClick={() => handleClick()} width={14} height={14} color='#353c52' />
+      <Info onMouseLeave={() => handleMouseLeave()} onClick={() => handleClick()} width={14} height={14} color='#353c52' />
```
Comment (G-Ray) [inline src/components/PasswordStrength/PasswordStrength.jsx:50]: :+1:

**Review by G-Ray — APPROVED** [review]

**Pull request merged**